testsuite/classlibrary/TestEnvGen_server.sc

```diff
@@ -27,6 +27,11 @@ TestEnvGen_server : UnitTest {
 			EnvGen.kr(Env([0, 1], [0.1]), doneAction: 2);
 		}.play(server);
 
+		// EnvGen in initialized by setting the state to the end of the Env,
```

Comment:
EnvGen in initialized –> EnvGen is initialized ?

Reply:
Thanks for spotting this! Fixed